conversion_functions.py

```diff
@@ -1,9 +1,4 @@
 # %%
```

Comment:
why are the imports omitted?

Reply:
Because they are no longer used. As I said, I have tried to make everything front-facing as independent of existing libraries as possible. Instead, I have moved everything to external files or libraries (such as `helpers.py`, `background.py`). The idea was that, in this way, we make gains in either modularity or maintainability (or ideally both).

Reply:
if you tested it locally and it works - everything is fine by me. we can see what @marcobonici  says after a review

thanks a lot, matthijs